flink-test-utils-parent/flink-connector-test-utils/pom.xml

```diff
@@ -54,6 +54,12 @@
 			<scope>compile</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.assertj</groupId>
```

Comment:
Is this the right place to add assertj dependency ?

Reply:
You don't need to change the pom, assertj should be ready to use in top level pom already. Please see https://github.com/apache/flink/pull/17871

Reply:
Looks this is a special module, as if I remove the dependency the dependency will not be found. Similarly in this pom.xml it declared `org.junit.jupiter:junit-jupiter` set and in parent pom it has `org.junit.jupiter:junit-jupiter` as well. So I figured this dependency is necessary here.

Reply:
The assertj with compile scope has been defined in the sibling module flink-test-utils-junit. Would you depend on it? We are preparing an official JUnit 5 upgrade process and will restructure the dependencies.

Reply:
I got your point, fair enough. We can do it for now and make an update when the upcoming official JUnit 5 upgrade process starts.